cli/src/base_cli/commands/register_tcb_info.rs

```diff
@@ -123,7 +125,7 @@ impl RegisterTcbInfoCommand {
 					chain_api.extrinsic_params(nonce)
 				);
 				nonce += 1;
-				match chain_api.submit_and_watch_extrinsic_until_success(xt, false) {
+				match chain_api.submit_and_watch_extrinsic_until(xt, XtStatus::InBlock) {
```

Comment:
until_success had the advantage that the user gets a report in the end and sees if any of the extrinsics have failed. why did you change that behavior here?

Reply:
you seem to favor this variant in other places too. I actually think we should use `until_success` by default and only use `until` if extrinsic failure is acceptable

Reply:
`*__until_sucess` doesn't exist anymore, and until success, with the `false` argument does exactly the same as the `*until` with the argument `InBlock`. I simply preserved the behaviour here.

Reply:
In block does also return the extrinsic report, but already does so when the extrinsic is in a block instead of waiting for that block to be finalized.